homeassistant/components/rfxtrx/binary_sensor.py

```diff
@@ -104,36 +111,28 @@ def binary_sensor_update(event):
         if not isinstance(event, rfxtrxmod.ControlEvent):
             return
 
-        device_id = slugify(event.device.id_string.lower())
-
-        sensor = RFX_DEVICES.get(device_id, get_pt2262_device(device_id))
+        data_bits = _get_device_data_bits(event.device, device_bits)
 
-        if sensor is None:
-            # Add the entity if not exists and automatic_add is True
-            if not config[CONF_AUTOMATIC_ADD]:
-                return
+        device_id = get_device_id(event.device, data_bits=data_bits)
+        if device_id in device_ids:
+            return
+        device_ids.append(device_id)
+
+        device_name = "-".join(device_id)
+        _LOGGER.info(
```

Comment:
We prefer debug level for this.

Reply:
Right. I am still pondering if there is much use for this separate pull . I might go for a config flow import style version directly. But I've not decided yet.